docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
+. Click *Close*.
 
-.. Drag and drop *Records* to the *Vertical axis* field.
+. Click *Save and return*.
 
-.. Drag and drop *@timestamp* to the *Horizontal axis* field.
+. Arrange the table panel so that it has its own row, but keep the default width of half the screen.
 
-. From the *Chart Type* dropdown, select *Percentage bar*.
-
-. To remove the vertical axis label, click *Left axis*, then deselect *Show*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_4_3.png[Turn off axis name]
+. You do not need a panel title because the table columns are clearly labeled.
 
 [discrete]
-[[add-the-response-code-filters]]
-==== Add the response code filters
-
-For each response code that you want to display, create a filter. 
-
-. In the editor, click the *Drop a field or click to add* field for *Break down by*.
-
-. From *Select a function*, click *Filters*.
-
-. Add the filter for the successful response codes. 
-
-.. Click *All records*.
-
-.. In the *KQL* field, enter `response.keyword>=200 AND response.keyword<300`. 
-
-.. In the *Label* field, enter `2XX`, then press Return.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_4_1.png[First filter in filters aggregation]
-
-. Add the filter for the redirect codes. 
-
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=300 AND response.keyword<400`. 
-
-.. In the *Label* field, enter `3XX`, then press Return.
-
-. Add the filter for the client error codes. 
-
-.. Click *Add a filter*.
-
-.. In the *KQL* field, enter `response.keyword>=400 AND response.keyword<500`. 
-
-.. In the *Label* field, enter `4XX`, then press Return.
+[[custom-ranges]]
+=== Compare the percentage of a subset to the whole
 
-. Add the filter for the server error codes. 
+To compare a subset to the whole you need to create non-overlapping sets of documents that add up to 100%.
+For numeric data the *intervals* function is the best option, while for all other data types *filters* is
+recommended. Use a proportion chart to display the values as a percentage of the sum. Lens has 5 types of proportion charts:
+pie, donut, treemap, percentage bar and percentage area.
 
-.. Click *Add a filter*.
+To determine if your users transfer more small files versus large files, create two non-overlapping intervals. Start
+with a bar chart and then switch to a pie chart to display as percentage:
 
-.. In the *KQL* field, enter `response.keyword>=500 AND response.keyword<600`. 
+. From the *Available fields* list, drag and drop `bytes` to *Vertical axis* in the editor.
 
-.. In the *Label* field, enter `5XX`, then press Return.
+. Click *Median of bytes* and then select *Sum* instead. Then *Close*.
 
-. To change the color palette, select *Status* from the *Color palette* dropdown, then click *Close*.
+. From the *Available fields* list, drag and drop `bytes` to *Break down by* in the editor, then specify the file size ranges.
```

Comment:
What are the file size ranges?

Reply:
Looks like this was accidentally deleted.